### ISSUE TITLE
consider allowing non-neutral stability with ninjafoam

### ISSUE THREAD

**Issue by nwagenbrenner** [firelab/windninja#107]
We could possibly do this in the same way we simulate diurnal winds with ninjafoam. Basically, do a ninjafoam simulation then use that output to do a native solver simulation with stability on. I guess I'm not sure how good or useful that might be since we'd be starting with a flow field that is already mass-consistent (at least for the domain used in ninjafoam) and aren't adding any "new flow" like we do in diurnal. What if we ran the native solver at a higher resolution? I'm still not sure this would be very realistic, but it might be worth trying?

**Comment by jforthofer** [comment]
I agree that it could be done, but I don't know that it would add much, since the flow out of OpenFOAM run should already conserve mass well.

**Comment by nwagenbrenner** [comment]
Okay, that's pretty much what I was saying. Just thought I'd throw it out there in case you had another perspective. We can look into simulating this effect later on with a buoyancy-driven flow solver in OpenFOAM (e.g., buoyantBoussinesqSimpleFoam).

**Comment by nwagenbrenner** [comment]
I was thinking about this again and realized we could do this if we use both the diurnal and stability parameterizations. So with the CFD option, non-neutral stability couldn't be used by itself, but it could be used with diurnal. I think we should consider this. 

In fact, after what we've seen recently with diurnal + stability (especially under stable conditions), I wonder if we shouldn't just combine them into one and have a single thermal parameterization? I think stability is best used with diurnal anyway and it would be simpler for users (less options).
